components/infobox/commons/infobox_person.lua

```diff
@@ -72,7 +72,7 @@ function Person:createInfobox()
 	end
 
 	local widgets = {
-		Header{name = self:nameDisplay(args), image = args.image, imageDefault = args.default},
+		Header{name = self:nameDisplay(args), image = args.image, imageDefault = args.default, subHeader = args.localid},
```

Comment:
If `name` is wiki customizable, I think the subHeader should be that as well

Reply:
Don't want to "risk" that a wiki feels the need to overwrite the Header entirely.

Reply:
header widget in this module is not overwritable :P
but i see the point, will adjust when back on pc later today